pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -2,207 +2,643 @@
-7. Setup the environment variables listed in the section below.
-
-### Environment variables
-
-These contain various settings used by the bot.
-To learn how to set environment variables, read [this page](../configure-environment-variables) first.
+If you don't wish to use the provided `config.yml` above, you can copy the `config-default.yml` file, rename it, and change whatever values you need. Note that you don't need to specify all items in `config.yml`, just the ones you want to override such as channel ID's. 
 
-The following is a list of all available environment variables used by the bot:
+See [here](../obtaining-discord-ids) for help with obtaining Discord ID's.
 
-| Variable | Required | Description |
-| -------- | -------- | -------- |
-| `BOT_TOKEN` | Always | Your Discord bot account's token (see [Test server and bot account](#test-server-and-bot-account)). |
-| `BOT_API_KEY` | When running bot without Docker | Used to authenticate with the site's API. When using Docker to run the bot, this is automatically set. By default, the site will always have the API key shown in the example below. |
-| `BOT_SENTRY_DSN` | When connecting the bot to sentry | The DSN of the sentry monitor. |
-| `BOT_TRACE_LOGGERS ` | When you wish to see specific or all trace logs | Comma separated list that specifies which loggers emit trace logs through the listed names. If the ! prefix is used, all of the loggers except the listed ones are set to the trace level. If * is used, the root logger is set to the trace level. |
-| `REDIS_PASSWORD` | When not using FakeRedis | The password to connect to the redis database. *Leave empty if you're not using REDIS.* |
-
----
+These are the main sections in `config-default.yml` that need overriding:
 
-If you are running on the host, while not required, we advise you set `use_fakeredis` to `true` in your `config.yml` file during development to avoid the need of setting up a Redis server.
-It does mean you may lose persistent data on restart but this is non-critical.
-Otherwise, you should set up a Redis instance and fill in the necessary config.
-{: .notification .is-warning }
+* `guild.id`
+* `guild.categories`
+* `guild.channels`
+* `guild.roles`
+* `guild.webhooks`
+* `style.emojis`
 
----
+Additionally:
 
-Example `.env` file:
+* Set `urls.site_schema` and `urls.site_api_schema` to `"http://"`.
+* Set `urls.site_api` to `!JOIN ["api.", *DOMAIN]`.
+* Set `urls.snekbox_eval_api` to `"http://localhost:8060/eval"`.
+* At this stage, set `bot.redis.use_fakeredis` to `true`. If you're looking for instructions for working with Redis, see [Working with Redis](#optional-working-with-redis).
 
-```shell
-BOT_TOKEN=YourDiscordBotTokenHere
-BOT_API_KEY=badbot13m0n8f570f942013fc818f234916ca531
-REDDIT_CLIENT_ID=YourRedditClientIDHere
-REDDIT_SECRET=YourRedditSecretHere
-```
+We understand this is tedious and are working on a better solution for setting up test servers.
```

Comment:
I don't think we need this line. We've had it for awhile now and no progress to the bootstrapping process has been made. I'd opt to remove it until we actually have a better way of getting this setup.

Reply:
We *are* working on a solution though :P

Reply:
We've been working on a solution for like... a year though >_>

Reply:
Still, I'd rather people didn't think we're not aware of this.